src/lib/perceivedLatencyInstrumentation.js

```diff
@@ -0,0 +1,96 @@
+/* @flow */
```

Comment:
I think this repo uses kabob-case for file names instead of camelCase. Can you rename this file to `preceived-latency-instrumentation.js`?